auslib/blobs/apprelease.py

```diff
@@ -908,6 +916,61 @@ def getReferencedReleases(self):
         return referencedReleases
 
 
+class AdditionalPatchAttributesXMLMixin(object):
+
+    def _getAdditionalPatchAttributes(self, patch):
+        additionalPatchAttributes = {'proof': ''}
+
+        if 'proof' in patch:
+            additionalPatchAttributes['proof'] = ' proof="%s"' % patch["proof"]
```

Comment:
What I meant here was to return something like:
{"proof": "abcdef123456789"}

...and let _getSpecificPatchXML iterate over them and construction the strings. If you want to return preconstructed strings here that's OK, but there's no point in doing so inside of a dictionary.